Library/Homebrew/env_config.rb

```diff
@@ -327,6 +327,22 @@ module EnvConfig
                      "from running `brew install` <formula> or `brew upgrade` <formula>.",
         boolean:     true,
       },
+      HOMEBREW_NO_UPDATE_REPORT_FORMULAE:    {
+        description: "If set, `brew update` will not show the list of newly added formulae.",
+        boolean:     true,
+      },
+      HOMEBREW_NO_UPDATE_REPORT_CASKS:    {
+        description: "If set, `brew update` will not show the list of newly added casks.",
+        boolean:     true,
+      },
+      HOMEBREW_NO_UPDATE_REPORT_FORMULAE_DELETED:    {
+        description: "If set, `brew update` will not show the list of deleted formulae.",
+        boolean:     true,
+      },
+      HOMEBREW_NO_UPDATE_REPORT_CASKS_DELETED:    {
+        description: "If set, `brew update` will not show the list of deleted casks.",
+        boolean:     true,
+      },
```

Comment:
```suggestion
      HOMEBREW_NO_UPDATE_REPORT_NEW:    {
        description: "If set, `brew update` will not show the list of newly added formulae or casks.",
        boolean:     true,
      },
```

(may need to run `brew style --fix` and split the string if it's too long)